Blog/templates/Blog/signup.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base_generic.html' %}
+{% load i18n %}
+
+{% block content %}
+    <div id="login-container" class="row">
+        <div id="login-panel" class="panel cell-lg-6 offset-lg-3 cell-md-8 offset-md-1">
+            <h1>{% trans 'Sign up' %}</h1>
```

Comment:
You marked this string as translatable, but the translation has not been updated.